scipy/signal/_bsplines.py

```diff
@@ -157,7 +156,20 @@ def thefunc(x):
 
 
 def bspline(x, n):
-    """B-spline basis function of order n.
+    """
+    .. deprecated:: 1.11.0
+
+        `scipy.signal.bspline` is deprecated in SciPy 1.11 and will be
+        removed in SciPy 1.13.
+
+        The exact equivalent (for a float array `x`) is
+
+        >>> from scipy.interpolate import BSpline
+        >>> knots = np.arange(-(n+1)/2, (n+3)/2)
+        >>> out = BSpline.basis_element(knots)(x)
+        >>> out[(x < knots[0]) | (x > knots[-1])] = 0.0
```

Comment:
Is there some directive or something I can use here so this code doesn't execute?

Reply:
Why is any of this needed?

Reply:
I mean, either the code snippet is incorrect and should be fixed (but why, what is the change that made it invalid?) or it is correct and the error is within the refguide-checker (which I'm happy to take a look at)

Reply:
Ah, we now require explicit `>>> import numpy as np` in all examples, and this one is missing it. Also in `cubic` and `quadratic`.

Reply:
Also `x` is not declared

Reply:
Thanks Jake. In that case, indeed, this is not meant to be a complete example, so OK to hide from the doctester in whichever way works :-).